@marsam If @toon23 rebases on the latest master (see https://github.com/NixOS/nixpkgs/pull/66154) will `profanity` build on `darwin`?

> ../Libraries/libcar/Sources/Rendition.cpp:29:10: fatal error: 'compression.h' file not found
#include <compression.h>

--- a/pkgs/applications/networking/instant-messengers/profanity/default.nix
+++ b/pkgs/applications/networking/instant-messengers/profanity/default.nix
@@ -20,13 +20,13 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "profanity-${version}";
-  version = "0.6.0";
+  version = "0.7.0";
 
   src = fetchFromGitHub {
-    owner = "boothj5";
+    owner = "profanity-im";
     repo = "profanity";
     rev = "${version}";
-    sha256 = "0f5kfzy22wzyj7rnd2nbj93q96ga87b53wlg8lfg83qdphx1ymz9";
+    sha256 = "15adg7ndjkzy04lizjmnvv0pf0snhzp6a8x74mndcm0zma0dia0z";
   };
 
   patches = [ ./patches/packages-osx.patch ./patches/undefined-macros.patch ];

--- a/pkgs/applications/networking/instant-messengers/profanity/default.nix
+++ b/pkgs/applications/networking/instant-messengers/profanity/default.nix
@@ -8,6 +8,7 @@
 , traySupport ? true,           gnome2 ? null
 , pgpSupport ? true,            gpgme ? null
 , pythonPluginSupport ? true,   python ? null
+, omemoSupport ? true,          libsignal-protocol-c ? null, libgcrypt ? null
 }:
 
 assert autoAwaySupport     -> libXScrnSaver != null && libX11 != null;
@@ -15,6 +16,7 @@ assert notifySupport       -> libnotify != null && gdk-pixbuf != null;
 assert traySupport         -> gnome2 != null;
 assert pgpSupport          -> gpgme != null;
 assert pythonPluginSupport -> python != null;
+assert omemoSupport        -> libsignal-protocol-c != null && libgcrypt != null;
 
 with stdenv.lib;
 
@@ -44,14 +46,16 @@ stdenv.mkDerivation rec {
     ++ optionals notifySupport       [ libnotify gdk-pixbuf ]
     ++ optionals traySupport         [ gnome2.gtk ]
     ++ optionals pgpSupport          [ gpgme ]
-    ++ optionals pythonPluginSupport [ python ];
+    ++ optionals pythonPluginSupport [ python ]
+    ++ optionals omemoSupport        [ libsignal-protocol-c libgcrypt ];
 
   # Enable feature flags, so that build fail if libs are missing
   configureFlags = [ "--enable-c-plugins" "--enable-otr" ]
     ++ optionals notifySupport       [ "--enable-notifications" ]
     ++ optionals traySupport         [ "--enable-icons" ]
     ++ optionals pgpSupport          [ "--enable-pgp" ]
-    ++ optionals pythonPluginSupport [ "--enable-python-plugins" ];
+    ++ optionals pythonPluginSupport [ "--enable-python-plugins" ]
+    ++ optionals omemoSupport        [ "--enable-omemo" ];
 
   preAutoreconf = ''
     mkdir m4

